packages/nimble-components/src/rich-text/viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -389,4 +389,57 @@ describe('RichTextViewer', () => {
             ).toEqual(['John Doe', 'Mary Wilson']);
         });
     });
+
+    describe('getMentionedHref() for viewer mentions', () => {
+        beforeEach(async () => {
+            ({ element, connect, disconnect } = await setup());
+            pageObject = new RichTextViewerPageObject(element);
+            await connect();
+        });
+
+        afterEach(async () => {
+            await disconnect();
+        });
+
+        it('getMentionedHref() method should return the mentioned href when it valid mention configuration matching the pattern to mention node', async () => {
```

Comment:
`it` refers to the thing specified in the `describe` string, i.e. "getMentionHref() for viewer mentions". So you don't need to put "getMentionedHref() method" in each of these names. The intent is for the test cases to read like, "`it` should return..."